package.json

```diff
@@ -169,26 +169,9 @@
       },
       {
         "permissionName": "ui-orders.receiving",
-        "displayName": "Orders: Receive items",
+        "displayName": "Orders: Receive/Check-in items",
         "description": "",
-        "visible": true,
-        "subPermissions": [
```

Comment:
where is subpermissions?

Reply:
what sub perms should I have for just button?